default.nix

```diff
@@ -4,7 +4,16 @@ stdenv.mkDerivation {
   name = "CasterSoundboard";
   src = ./CasterSoundboard;
   nativeBuildInputs = [ qt5.qmakeHook qt5.makeQtWrapper ];
-  buildInputs = [ qt5.qtmultimedia ];
+  buildInputs = [
+    qt5.qtmultimedia
+    gst_all_1.gst-plugins-base
```

Comment:
This makes sense since [gstreamer is the qtmultimedia backend](https://wiki.qt.io/Qt_5.7_Multimedia_Backends) on the unixes